Tasks/VsTestPlatformToolInstallerV1/task.json

```diff
@@ -110,21 +110,21 @@
     },
     "messages": {
       "StartingInstaller": "Starting VsTest platform tools installer task.",
-      "ProvideExplicitVersion": "The version you provided (%s) is not an explicit version. Please provide an explicit version number.",
+      "ProvideExplicitVersion": "The version you provided (%s) is not an explicit version. Please provide a version number of the form majorVer.minorVer.patchVer.",
```

Comment:
get this message reviewed with prachi.

Reply:
will do. She said she'll review some time this week